eng/targets/Release.props

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>1.2.2</VersionPrefix>
+    <VersionPrefix>1.2.3</VersionPrefix>
```

Comment:
I wonder this should be v1.2.3 or v1.3.0. Since [PR#290](https://github.com/microsoft/durabletask-dotnet/pull/290/files) seems like a new feature.